src/components/UnbraceThemeProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { ThemeProvider } from 'styled-components';
+import merge = require('lodash.merge');
+import { componentTheme, ComponentTheme } from '../theme';
+
+type Props = {
+  children: React.ReactElement;
+  customTheme?: Partial<ComponentTheme>;
```

Comment:
This typing is too restricting for the user. This only allows for their own custom theme to be a partial of our components theme. They should be allowed to have whatever theme they want

Reply:
fixed

Reply:
This still isn't right